papers/ring_vrf/zk_cont.tex

```diff
@@ -1,34 +1,101 @@
-
-\section{Zero-knowledge continuations}
+\section{Zero-knowledge Continuations}
 \label{sec:rvrf_cont}
 
-We now develop our zero-knowledge continuation technique, called 
-{\em special(ized) Groth16} or \SpecialG, and
- build ring VRFs with fast amortized prover time.
-Anonymity demands we finalize the amortized ``continuation'' multiple
-times, with each time being unlinkable to the others, meaning our rVRF
-stays zero-knowledge even with the continuation being reused.
-
-Our zero-knowledge continuation $\SpecialG_\rel$ consists of three algorithms:
+\noindent In the following, we describe a NIZK for a relation $\rel$ where
+$$\rel = \{(\bary, \barz; \barx, \baromega, \baromegap):  (\bary, \barx; \baromega) \in \relone, (\barz, \barx; \baromegap) \in \reltwo \},$$
+and $\relone$, $\reltwo$ are some NP relations. Our NIZK is designed to efficiently re-prove membership for relation $\relone$
+via a new technique which we call \emph{zero-knowledge continuation}. In practice, using a NIZK that is a zero-knowledge continuation 
+ensures one essentially needs to create only once an otherwise expensive proof for $\relone$ which can later be 
+re-used multiple times (just after inexpensive re-randomisations) while preserving knowledge soundness and zero-knowledge. 
+Below, we formally define zero-knowledge continuation. In section~\ref{sec:rvrf_groth16} we instantiate it via a \emph{special(ized) 
```

Comment:
We can add in an appropriate place a sentence to show how we suggest prove \rel with zkcont in the end i.e,
In the end, we complete the proof for \rel together with zkcont proof on \relone and an NIZK proof on \reltwo.